opds.py

```diff
@@ -653,7 +653,7 @@ def page(cls, _db, title, url, lane, annotator=None,
             title = visible_parent.display_name
         else:
             title = top_level_title
-        up_uri = annotator.groups_url(visible_parent)
+        up_uri = annotator.lane_url(visible_parent)
```

Comment:
The content_server doesn't have any grouped feeds, so it makes more sense to use lane_url. In the circulation manager, lane_url can return a grouped feed url or a page feed url depending on whether the lane has visible sublanes.